engine/settings/base.py

```diff
@@ -67,6 +67,7 @@
 FEATURE_GRAFANA_ALERTING_V2_ENABLED = getenv_boolean("FEATURE_GRAFANA_ALERTING_V2_ENABLED", default=False)
 GRAFANA_CLOUD_ONCALL_HEARTBEAT_ENABLED = getenv_boolean("GRAFANA_CLOUD_ONCALL_HEARTBEAT_ENABLED", default=True)
 GRAFANA_CLOUD_NOTIFICATIONS_ENABLED = getenv_boolean("GRAFANA_CLOUD_NOTIFICATIONS_ENABLED", default=True)
+TELEGRAM_LONG_POLLING_ENABLED = getenv_boolean("TELEGRAM_LONG_POLLING_ENABLED", default=False)
```

Comment:
I think you missed to rename the setting here? (it should be `FEATURE_TELEGRAM_LONG_POLLING_ENABLED` instead, right? and the env variable name as well)

Reply:
You're right. Fixed name, and moved it up next to `FEATURE_TELEGRAM_INTEGRATION_ENABLED` setting